model/task/task.go

```diff
@@ -2884,23 +2884,29 @@ func addStatusColorSort(key string) bson.M {
 							},
 							"then": 1, // red
 						},
+						{
+							"case": bson.M{
+								"$in": []interface{}{"$" + key, []string{evergreen.TaskKnownIssue}},
```

Comment:
evergreen.TaskKnownIssue seems to be undefined from the logs: https://evergreen.mongodb.com/lobster/evergreen/task/evergreen_ubuntu1604_test_graphql_patch_4071b3c5984744e3a71589c95445d89f6db446be_60ad1ca52a60ed4e293079bf_21_05_25_15_49_58/1/task#bookmarks=0%2C204&l=1&shareLine=178

Reply:
it relies on https://github.com/evergreen-ci/evergreen/pull/4673